react/Textarea/Textarea.js

```diff
@@ -15,6 +16,7 @@ function combineClassNames(props = {}, ...classNames) {
   };
 }
 
+/* eslint-disable react/no-deprecated, react/no-danger */
```

Comment:
Not sure there's anything to ignore here 🤔 no-danger could be moved to respective line